superset/config.py

```diff
@@ -12,8 +12,8 @@
 # software distributed under the License is distributed on an
 # "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 # KIND, either express or implied.  See the License for the
-# specific language governing permissions and limitations
-# under the License.
+# specific language governing permissions and limitations under
+# the License.
```

Comment:
Maybe this change was done unintentionally?

Reply:
oh I needed to trigger the build... let me change it back